app/controllers/content_items_controller.rb

```diff
@@ -22,6 +22,11 @@ def show
 
 private
 
+  def layout
+    paths = %w[service-manual service-toolkit]
+    paths.include?(params[:path]) ? "gem_layout_full_width_no_footer_navigation" : "gem_layout_no_footer_navigation"
+  end
+
```

Comment:
Not sure if I'm just being dense here, but isn't this only going to work for pages where the path is exactly "service-manual" or "service-toolkit"? It looks like service-toolkit doesn't have any sub-pages, but service-manual does. Is this going to work for service-manual/design? I assume params[:path] has that whole block in it? Or am I wrong?

Reply:
That's correct this will only work for pages where the path is exactly "service-manual" or "service-toolkit" (and the `gem_layout_full_width_no_footer_navigation` should only be used on the landing pages for the Service Manual - https://www.gov.uk/service-manual and https://www.gov.uk/service-toolkit).

It looks that way to me too that only the service manual has sub pages (and should use the `gem_layout_no_footer_navigation` layout).

I tested with /service-manual/helping-people-to-use-your-service and the correct template did seem to be used (screenshot included above).

Reply:
So in the third screenshot pair (where there's no big blue header), it's correct that they are identical? (the small blue bar appears in both)

Reply:
Yep that's right. It's essentially included to show there's been no visual regressions on the sub pages because of these changes.